variation/to_vrs.py

```diff
@@ -201,23 +105,57 @@ async def to_vrs(self, q: str,
             unable to translate or normalize query.
         :return: ToVRSService containing VRS variations and warnings
         """
-        validations, warnings = await self.get_validations(q)
-        translations, warnings = self.get_translations(validations, warnings)
+        warnings = []
+        variations = []
+        params = {
+            "search_term": q,
+            "variations": variations,
+            "service_meta_": ServiceMeta(
+                version=__version__, response_datetime=datetime.now()
+            ),
+            "warnings": warnings,
+        }
+
+        # Get tokens for input query
+        tokens = self.tokenizer.perform(unquote(q.strip()), warnings)
+        if warnings:
+            params["warnings"] = warnings
+            return ToVRSService(**params)
+
+        # Get classification for list of tokens
+        classification = self.classifier.perform(tokens)
+        if not classification:
+            params["warnings"] = [f"Unable to find classification for: {q}"]
+            return ToVRSService(**params)
+
+        # Get validation summary for classification
+        validation_summary = await self.validator.perform(classification)
+        if validation_summary:
+            # Get translated VRS representation for valid results
+            translations, warnings = await self.get_translations(
+                validation_summary,
+                warnings,
+                endpoint_name=Endpoint.TO_VRS,
+                hgvs_dup_del_mode=HGVSDupDelModeOption.DEFAULT,
+                do_liftover=False,
+            )
+        else:
+            translations = []
```

Comment:
I think this is unreachable -- an empty ValidationSummary evaluates to True

```
In [4]: bool(ValidationSummary(valid_results=[], invalid_results=[], warnings=[]))
Out[4]: True
```

Reply:
I need to change it to `if validation_summary.valid_results` and update `get_translations` accordingly. I think is cleaner